src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1507,13 +1507,13 @@
 					self._getItemsFromElement();
 					self._createTextInputs();
 
-					ui.arcListviewSelection = self._buildArcListviewSelection(page);
-					arcListviewCarousel = buildArcListviewCarousel(carousel, visibleItemsCount);
-					ui.arcListviewCarousel = arcListviewCarousel;
+					ui.arcListviewSelection = self._buildArcListviewElement(page, classes.SELECTION);
+					ui.arcListviewCarousel = buildArcListviewCarousel(carousel, visibleItemsCount);
+					ui.arcListviewCarouselDummyElement = self._buildArcListviewElement(page, classes.CAROUSEL_DUMMY);
```

Comment:
The name of properties can be shorter. 
eg. `ui.dummyElement` will be fine.

Reply:
Ok, used ui.dummyElement